src/pages/Sketch/components/Canvas.jsx

```diff
@@ -3,13 +3,16 @@ import styles from "./canvas.module.scss";
 import Toolbox from "./Toolbox/Toolbox";
 import { FaDownload, FaStar } from "react-icons/fa";
 import { VscSaveAs } from "react-icons/vsc";
+import { BsBoxArrowInDown } from "react-icons/bs";
 import { RiDeleteBinLine } from "react-icons/ri";
 import { AiOutlineFolderOpen } from "react-icons/ai";
 import IconsLibrary from "./IconLibrary/IconsLibrary";
 import ReactSnackBar from "react-js-snackbar";
 import checkBox from "./../../../assets/images/checkmark.svg";
 import rough from "roughjs/bin/rough";
 
+
```

Comment:
Avoid these extra spaces as it might lead to merge conflicts in someone else's PR

Reply:
sure! i will keep this in mind. thanku